ProjectCoimbra.UWP/DataAccessLibrary/DataAccess.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed under the MIT License.
+
+namespace DataAccessLibrary
```

Comment:
I'd suggest `Coimbra.DataAccess`. It aligns with the other project namespaces, and it doesn't include `Library` which I think is unnecessary given that essentially everything is a library in .NET